packages/react-components/src/components/Picker/Picker.stories.tsx

```diff
@@ -111,12 +110,49 @@ PickerWithDisabledSearch.args = {
     { key: 'six', name: 'Option six', disabled: true },
     { key: 'seven', name: 'Option seven', disabled: true },
   ],
-  onSelect: (item) => console.log(item),
 };
 
 export const PickerWithSelectedOption = StoryTemplate.bind({});
 PickerWithSelectedOption.args = {
-  selectedOption: { key: 'two', name: 'Option two' },
+  selectedOptions: [{ key: 'two', name: 'Option two' }],
+  options: [
+    { key: 'groupA', name: 'Group A title header', groupHeader: true },
+    { key: 'one', name: 'Option one' },
+    { key: 'two', name: 'Option two' },
+    { key: 'three', name: 'Option three' },
+    { key: 'groupB', name: 'Group B title header', groupHeader: true },
+    { key: 'four', name: 'Option four' },
+    { key: 'five', name: 'Option five' },
+    { key: 'six', name: 'Option six', disabled: true },
+    { key: 'seven', name: 'Option seven', disabled: true },
+  ],
```

Comment:
Have you considered moving to a common variable, or do we think it won't work well in Storybook docs?